code/datums/martial.dm

```diff
@@ -386,19 +389,22 @@
 	usr << "<span class='notice'>Head Kick</span>: Disarm Harm Harm. Decent damage, forces opponent to drop item in hand."
 	usr << "<span class='notice'>Elbow Drop</span>: Harm Disarm Harm Disarm Harm. Opponent must be on the ground. Deals huge damage, instantly kills anyone in critical condition."
 
-//CQC
+//CQC _bman_
```

Comment:
sigs can be useful so you know who to yell at when something is borked, but i guess if that's how we do it then sur.